VideoWeb/VideoWeb/ClientApp/src/app/vh-officer/participant-status/participant-status.component.html

```diff
@@ -4,7 +4,20 @@
 
 <div id="participant-status-list">
   <div *ngFor="let participant of participants" class="govuk-grid-row">
-    <div class="govuk-grid-column-one-half">
+    <div class="govuk-grid-column-one-half status-box-display">
+      <div class="interpreter-icon">
+        <span *ngIf="participant.isInterpreterOrInterpretee" class="vh-interpreter-icon">
+          <em class="far fa-2x fa-language"></em>
```

Comment:
i think should use span, is this a code smell?

Reply:
changed to span